Maquetacion/scss/styles.scss

```diff
@@ -0,0 +1,4 @@
+.title {
+    color: red;
+}
+  
```

Comment:
Remember that the last line should be empty. There seems to be some spaces here.

Reply:
👍